doorstop/core/vcs/test/test_commands.py

```diff
@@ -79,44 +85,7 @@ def test_commit(self, mock_call):
 
 
 @patch('subprocess.call')  # pylint: disable=R0904
-class TestMockVCS(BaseTestCase):
-    """Tests for the placeholder VCS plugin."""
```

Comment:
BaseTestCase.test_missing_command will fail for TestMockVCS and this class never makes any calls. I didn't really see the point of TestMockVCS so I though it made more sense to remove it than to add a work around.